webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -207,7 +215,7 @@ export default class InteractiveDialog extends React.PureComponent {
                 >
                     <Modal.Header
                         closeButton={true}
-                        style={{borderBottom: elements == null && '0px'}}
+                        style={{borderBottom: elements == null ? '0px' : undefined}}
```

Comment:
This change is not necessarily wrong, but is unneeded, any reason to include it?

Reply:
![Uploading image.png…]()

Reply:
Ok, that makes sense :+1: